script/validate-data/index.ts

```diff
@@ -41,7 +42,7 @@ const propertiesSchema = {
 
 async function checkWorkflows(folders: string[]): Promise<WorkflowWithErrors[]> {
   const result: WorkflowWithErrors[] = []
-
+  const workflow_template_names : string[] = []
```

Comment:
Since this is going to be a unique, you can use a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)

Reply:
https://stackoverflow.com/questions/40180641/hashset-in-typescript